Automattic/wp-calypso
# Newsletter Flow: Plan step to use new grid

[Context](p9Jlb4-6L9-p2)

This PR updates the Newsletter flow to use a new design for the Plans step:

![image](https://user-images.githubusercontent.com/52076348/235179857-76a3c8e4-f1ad-40ae-9072-c1281b78dc35.png)

## Testing

1. Checkout the branch and run `yarn start` or use the Live Link
2. Visit `setup/newsletter` and reach the plans step
3. Verify that the step works correctly. Remember to check the plans comparison at the end of the step.

Fixes https://github.com/Automattic/wp-calypso/issues/75060
